Source/WebInspectorUI/UserInterface/Views/Sidebar.js

```diff
@@ -191,6 +208,52 @@ WI.Sidebar = class Sidebar extends WI.View
     {
         // Implemented by subclasses if needed.
     }
+    
+    sizeDidChange()
+    {
+        super.sizeDidChange();
+
+        if (this._side !== WI.Sidebar.Sides.Trailing || !this._canMoveToBottom)
+            return;
+        
+        let hasHeightResizer = this._heightResizer.element.parentNode;
+        
+        if (WI.layoutMode === WI.LayoutMode.Narrow && !hasHeightResizer) {
+            this.element.style.width = "";
+            this.element.insertBefore(this._heightResizer.element, this.element.firstChild);
```

Comment:
i think we should just always attach this to the DOM in the `constructor` and then hide/disable it using CSS
```
body:not(.narrow) .sidebar > .resizer.horizontal-rule {
    display: none;
}
```

and then in `SingleSidebar.css` you could do something similar
```
body.narrow .single-sidebar > .resizer.vertical-rule {
    display: none;
}
```

as far as `hasHeightResizer`, i think you could replace that with a `_didMoveToBottom` (similar to how you have a `_canMoveToBottom`) that changes based on the `WI.layoutMode`
```
if (WI.layoutMode === WI.LayoutMode.Narrow && !this._didMoveToBottom) {
    this._didMoveToBottom = true;
    this.element.style.width = "";
    this.dispatchEventToListeners(WI.Sidebar.Event.PositionDidChange);
} else if (WI.layoutMode !== WI.LayoutMode.Narrow && this._didMoveToBottom) {
    this._didMoveToBottom = false;
    this.element.style.height = "";
    this.dispatchEventToListeners(WI.Sidebar.Event.PositionDidChange);
}
```